src/main/java/com/zufar/onlinestore/payment/dto/CreatePaymentDto.java

```diff
@@ -0,0 +1,16 @@
+package com.zufar.onlinestore.payment.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.math.BigDecimal;
+
+public record CreatePaymentDto(
+        @NotBlank(message = "Payment method id is mandatory field")
+        String paymentMethodId,
```

Comment:
please think which modifier suits the most

Reply:
> please think which modifier suits the most

Done